swo_aws_extension/flows/fulfillment/pipelines.py

```
@@ -0,0 +1,70 @@
+    -> ejecuta el post-step
+
+
+
+
+
+"""
+
+
+# TODO This function is used to notify about unhandled exceptions in AWS pipelines.
+# Should be removed with the new SDK error handling mechanism.
+def pipeline_error_handler(error: Exception, context: Context, next_step):
```

Ruff (0.14.8)

Unused function argument: `next_step`

(ARG001)